main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -744,6 +745,54 @@ Resources:
       ManagedPolicyArns:
         - !Ref PolicyDataSourceReachabilityHandler
       PermissionsBoundary: !Ref PolicyDataSourceReachabilityHandler
+  
+  PolicyAwsAccountOnboardingHandler:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Description: Grant CloudFormation access to check CFN stack status and ddb access to onboard accounts
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Action:
+              - dynamodb:GetItem
+              - dynamodb:Query
+              - dynamodb:Scan
+              - dynamodb:UpdateItem
+            Resource:
+              - !GetAtt [AwsAccountsDb, Arn]
+              - !Join ['', [!GetAtt [AwsAccountsDb, Arn], '/index/*']]
+          - Effect: Allow
+            Action:
+              - dynamodb:GetItem
+            Resource: !GetAtt [WorkflowsDb, Arn]
+          - Effect: Allow
+            Action: dynamodb:UpdateItem
+            Resource: !GetAtt [WorkflowInstancesDb, Arn]
+          - Effect: Allow
```

Comment:
Double check the log is printed successfully with these permission? I remember it's a bit tricky restricting CloudWatch permissions.

Reply:
hmmm I don't think it is honestly, it was giving me some error message. I'll have to look at it again

Reply:
You can reference the example here -> https://github.com/awslabs/service-workbench-on-aws/blob/35f6cce3ccc301921ead742240c15c1a7e332f0c/addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/sagemaker-notebook-instance.cfn.yml#L86

Reply:
thanks! looks like I was missing two allowed actions, deploying it now to verify that everything is working

Reply:
everything is working properly now 👍🏻